vagrant/provisioning/roles/httpd/templates/ldap-auth-restricted-linuxadmins-helper.script2

```diff
@@ -0,0 +1,14 @@
+AuthType Basic
+AuthName "Restricted applications"
+AuthBasicProvider ldap
+RewriteEngine On
+#LDAPReferrals Off
+# Perform LDAP lookup for user to check group membership
+AuthLDAPUrl "{{ ldap_url }}/DC=appdev,DC=armedia,DC=com?sAMAccountName?sub?(objectClass=*)"
```

Comment:
The domain name **DC=appdev,DC=armedia,DC=com** should be configured in the facts file.

Reply:
I believe the entire LDAP URL is already a fact (ldap_url), you could just use this existing fact.  The ldap_base is also a fact already.  So you just have to use these two existing facts, versus defining a new one.